apps/storybook/src/Selection.stories.tsx

```diff
@@ -17,8 +17,6 @@ import FillHeight from './decorators/FillHeight';
 
 interface TemplateProps {
   selectionType: 'line' | 'rectangle';
-  disablePan?: boolean;
-  disableZoom?: boolean;
```

Comment:
I've chosen to always enable pan/zoom in the selection stories after all. It's annoying to have to enable them to test that it still works regardless of zoom level. I've also renamed the stories a bit.

Reply:
Doesn't it raise an issue when having no `modifierKey` for the selection ? Dragging when zoomed in could then trigger the selection and the pan.

Thanks for the renaming, that's way better !

Reply:
Arf, forgot about this, sorry.